docs/aws_vpc_subnet.md

```diff
@@ -0,0 +1,148 @@
+---
+title: About the aws_vpc_subnet Resource
+---
+
+# aws_vpc_subnet
+
+Use the `aws_vpc_subnet` InSpec audit resource to test properties of a vpc subnet.
+
+To test properties of a single VPC subnet, use the `aws_vpc_subnet` resource.
+
+To test properties of all or a group of VPC subnets, use the `aws_vpc_subnets` resource.
+
+<br>
+
+## Syntax
+
+An `aws_vpc_subnet` resource block uses the parameter to select a VPC and a subnet in the VPC.
+
+    describe aws_vpc_subnet(vpc_id: 'vpc-01234567', subnet_id: 'subnet-1234567') do
```

Comment:
I don't think you need to require vpc_id.  If someone already has the subnet ID, that's unique across the account, at least; carrying the VPC_ID adds inconvenience and does not add value.